tests/x11/firefox.pm

```diff
@@ -16,12 +16,24 @@ use base "x11test";
 use strict;
 use warnings;
 use testapi;
-use version_utils 'is_tumbleweed';
+use version_utils qw(is_opensuse is_tumbleweed);
 use Utils::Architectures 'is_s390x';
 
 sub run() {
     my ($self) = shift;
 
+    # we have sometime issue with firefox crash, export setting of automatic sending of crash report
```

Comment:
Did you miss `source ~/.profile` ? :)

Reply:
yes, the file is not there, I checked :)